### PR TITLE
Update frFR.lua

### PR DESCRIPTION
**Pull request by Alyndel**
Updated all translations in French for War Within, including Delves, Dungeons and Scenarios

### CODE REVIEW

File: Delves/TheSinkhole/Locales/frFR.lua [hunk 1]
```diff
@@ -8,5 +8,5 @@ L = BigWigs:NewBossLocale("Leviathan Caller", "frFR")
 if L then
 	L.leviathan_caller = "Cor du léviathan"
 	L.guardian_tentacle = "Tentacule gardien"
-	--L.slamming_tentacles = "Slamming Tentacles" -- plural
```
Comment (ntowle) [inline Delves/TheSinkhole/Locales/frFR.lua:11]: Is this right? It's a different NPC than the above, wowhead: https://www.wowhead.com/fr/npc=220741/tentacule-percutant

File: WarWithin/PrioryOfTheSacredFlame/Locales/frFR.lua [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
 	L.ardent_paladin = "Paladin ardent"
-	--L.risen_mage = "Risen Mage"
-	--L.sir_braunpyke = "Sir Braunpyke"
+	L.risen_mage = "Mage ressucité"
+	L.sir_braunpyke = "Baron Braunpique"
 
-	--L.baron_braunpyke_warmup_trigger = "They've served their purpose. Baron, demonstrate your worth."
-	--L.custom_on_autotalk_desc = "|cFFFF0000Requires Priest or Paladin.|r Automatically select the NPC dialog option that grants you the 'Blessing of the Sacred Flame' aura."
+	L.baron_braunpyke_warmup_trigger = "Leur mission est remplie. Baron, montrez votre valeur."
+	L.custom_on_autotalk_desc = "|cFFFF0000Requires Priest or Paladin.|r Sélectionne automatiquement le dialogue avec le PNJ qui octroie à votre groupe l'amélioration 'Bénédiction de la Flamme sacrée'."
```
Comment (ntowle) [inline WarWithin/PrioryOfTheSacredFlame/Locales/frFR.lua:21]: should `Requires Priest or Paladin.` be translated too?

Comment (Alyndel) [reply WarWithin/PrioryOfTheSacredFlame/Locales/frFR.lua]: Thanks, both of your requests have been translated. Some lines have gone through the net, sorry :)

**Comment by ntowle** [comment]
Great work, thanks so much.

There are also some additional untranslated French locale strings for War Within Season 1:
- https://github.com/BigWigsMods/LittleWigs/blob/master/BfA/SiegeOfBoralus/Locales/frFR.lua
- https://github.com/BigWigsMods/LittleWigs/blob/master/Shadowlands/TheNecroticWake/Locales/frFR.lua
- https://github.com/BigWigsMods/LittleWigs/blob/master/Cataclysm/GrimBatol/Locales/frFR.lua

**Pull request merged**